### PR TITLE
Some examples of IRC bot commands

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: docs/server/push-duty.rst [hunk 1]
```diff
@@ -99,10 +98,32 @@ Then update the etherpad with the new comparison link for the updated tag.
 Push to stage
 +++++++++++++
 
-Once the tag is ready and any needed docker containers have been built, it
+Once the tag is ready and any needed
+`Docker Hub`_ containers have been built, it
 needs to be pushed to the staging server, using the `IRC bot in #amopush
 <https://mana.mozilla.org/wiki/display/SVCOPS/AMO+Dev+Resources#AMODevResources-DeployingwithJenkinsBot>`_.
-Jason can help with that if needed.
+Consult the IRC bot docs for more up to date information but here are examples
```
Comment (tofumatt) [inline docs/server/push-duty.rst]: Can you link to those docs from here?

File: docs/server/push-duty.rst [hunk 2]
```diff
@@ -99,10 +98,32 @@ Then update the etherpad with the new comparison link for the updated tag.
...
+* ``!build admin-stage YYYY.MM.DD``
+* ``!build discopane-stage YYYY.MM.DD``
+* ``!build versioncheck-stage YYYY.MM.DD``
+
+Each of these apps live in the code respositories that are listed for tagging up
+above.
+
+When you run each command in ``#amopush`` you will soon see a
+``Job amo-xxxx-BUILD succeeded``
+message. Do not be fooled! This doesn't mean the change is live. You need to
```
Comment (tofumatt) [inline docs/server/push-duty.rst]: 👍

File: docs/server/push-duty.rst (hunk 1 above)
Comment (kumar303) [inline docs/server/push-duty.rst]: it's linked on the line above

Comment (tofumatt) [inline docs/server/push-duty.rst]: Haha so it is. I'm dumb, all good!

Comment (kumar303) [inline docs/server/push-duty.rst]: Well, it's hard to read ReST/Sphinx diffs. I think it should be clear in the HTML.

**Comment by tofumatt** [comment]
Agreed. 👍🏻

**Pull request merged**